docs/book/src/topics/powervs/prerequisites.md

```diff
@@ -9,8 +9,6 @@
 4. An appropriately configured [Go development environment](https://golang.org/doc/install)
 5. Install `clusterctl` tool (see [here](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl))
 6. Install `pvsadm` tool (see [here](https://github.com/ppc64le-cloud/pvsadm#installation))
-6. Install `ibmcloud` tool (see [here](https://github.com/IBM-Cloud/ibm-cloud-cli-release#downloads))
```

Comment:
I think we should mention how to get capibmadm cli tool as well.

Reply:
Fixed.